get_active_hooks.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""
+    Report on Service & Web hooks for organization.
+"""
+import argparse
+import client
+import logging
+import urlparse
+import yaml
+
+"""
+    Lore:   swapping hook.test for hook.ping will cause repetition of the
+            actions.  In particular, a number of repos post to IRC channels
+            and/or bugs on commits, so expect comments to that effect.
+"""
+
+logger = logging.getLogger(__name__)
+
+
+def report_hooks(gh, org, active_only, unique_only, do_ping, yaml_out):
```

Comment:
The occasional newline helps group things and improves readability. You don't have to add them now, just keep it in mind :)